packages/frontend-shared/src/components/CopyText.vue

```diff
@@ -0,0 +1,26 @@
+<template>
```

Comment:
Is this a component we should re-use elsewhere, such as in Project Settings?

Reply:
The design is slightly different there, but you're right, that settings page, and also the TerminalPrompt component are all kind of similar layouts of the same idea. Maybe it would be sensible to make one component with 3 variations. That's kinda why I didn't just want to do this markup right in the login modal, it seems like the kind of design element we'll want to reuse.